Chap_Terms.tex

```diff
@@ -138,9 +145,8 @@ \section{Naming Conventions}
 \item The \code{pmix_} prefix is used to denote function pointer and type definitions.
 \end{itemize}
 
-Users should not use the \textbf{\code{"PMIX"}}, \textbf{\code{"PMIx"}}, or \textbf{\code{"pmix"}} prefixes in their applications or libraries so as to avoid symbol conflicts with current and later versions of the \ac{PMIx} Standard.
+Users shall not use the \textbf{\code{"PMIX_"}}, \textbf{\code{"PMIx_"}}, or \textbf{\code{"pmix_"}} prefixes for symbols in their code so as to avoid symbol conflicts with \ac{PMIx} implementations.
 
```

Comment:
Since you are explicitly carving out these prefixes, should you also explicitly prohibit the use of string attributes that start with "pmix"? We state this in the chapter on "reserved keys", but given that you are placing a restriction on symbols here, it might fit to repeat the attribute reservation.

Reply:
We have a note for that a few lines up in this PR:
 * https://github.com/pmix/pmix-standard/pull/347/files#diff-cb249f8e90208e1194ad52d5e550966ceca13e78cd8512516575d7cd5bd970a4R139

> \item The string representation of attributes are prefixed with \code{"pmix"}.

Reply:
Understood - but my point was that if you explicitly tell users they cannot use prefixed symbols, then you probably also need to explicitly tell them they cannot use prefixed attributes. You already cited the prefixed symbol standard above as well, so you now are implying that even though there exist other rules (e.g., the prefixed attribute), only the symbols are protected.